runtime/src/qio/qio.c

```diff
@@ -2883,7 +2891,6 @@ qioerr _qio_buffered_read(qio_channel_t* ch, void* ptr, ssize_t len, ssize_t* am
       }
       // Return early on an error or on EOF.
       if( err ) {
-        *amt_read = len - remaining;
```

Comment:
Why is this line removed?

Reply:
This was a second minor bug that was invalidating `amt_read` when the file pointer is at EOF.